src/i18n/en-US.properties

```diff
@@ -12,6 +12,8 @@ enter_fullscreen=Enter fullscreen
 exit_fullscreen=Exit fullscreen
 # Button tooltip for annotation region comment
 region_comment=Comment on Region
+# Button tooltip for annotation highlight text
+highlight_text=Text Highlighter
```

Comment:
It seems like the copy here should be written as a directive rather than a description.

Region: Comment on Region
Highlight: Highlight text